packages/components/src/Link/Link.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { PropsWithChildren } from "react";
+
+interface LinkProps {
+  readonly ariaLabel?: string;
+  readonly ariaExpanded?: boolean;
+  readonly external?: boolean;
+  readonly url: string;
+}
+
+export function Link({
+  ariaLabel,
+  ariaExpanded,
+  external = false,
+  url,
+  children,
```

Comment:
Same small style suggestion: consider moving `url` and `children` up to the top of this list.